geocoder/src/main/scala/grasshopper/geocoder/search/census/AddressInterpolator.scala

```diff
@@ -89,8 +89,9 @@ object AddressInterpolator {
     } else if (!addressIsEven && !leftRangeIsEven) {
       "L"
     } else {
-      log.warn(s"Could not determine if range is even in feature: ${f.toString}")
-      "R"
+      val default = "R"
```

Comment:
Sorry ... but couldn't you just say `Defaulting to R` and then just return `"R"` like what was happening already?